src/tableFactory.ts

```diff
@@ -35,19 +35,24 @@ class TableValidator {
     }
 
     private static isSeparator(cellValue: string, isFirstOrLast: boolean): boolean {
-        if (cellValue.trim() == "-")
-            return true;
+        // if (cellValue.trim() == "-")

```

Comment:
Please don't leave commented code hanging around. In case history is needed, there is source control for it.